SU-SWS/decanter
# Updated card component to HTML5

# READY FOR REVIEW
This replaces `<div>` with `<article>` and `<section>`

# Summary
- Update Decanter to use HTML5

# Needed By (Date)
- soon

# Urgency
- Not urgent

# Steps to Test

1. Checkout this branch
2. Verify the code changes make sense
3. Verify the card component still works
4. Verify the rendered output (inspect) makes sense

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?
Decanter
# Associated Issues and/or People
- JIRA ticket - https://stanfordits.atlassian.net/browse/THEME-15
- Other PRs - https://github.com/SU-SWS/decanter/pull/421
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
